### PR TITLE
Updating the sp_execute_external_script known issues

### PR DESCRIPTION
**Pull request by monamaki**
This change covers performance issues of Process Pooling in ML Services (R and Python) . We discussed the cold start performance of Process Pooling in ML Services and provided a workaround for each potential relevant issue.

replaced by https://github.com/MicrosoftDocs/sql-docs-pr/pull/25933

### CODE REVIEW

**Comment by monamaki** [comment]
Hey @arunguru-msft, can you please review this change?

**Comment by monamaki** [comment]
Hey @uc-msft , can you please review this change?

**Comment by Jak-MS** [comment]
@WilliamDAssafMSFT 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/machine-learning/troubleshooting/known-issues-for-sql-server-machine-learning-services.md [hunk 1]
```diff
@@ -281,6 +281,29 @@ The best solution is to upgrade to [!INCLUDE[sssql19-md](../../includes/sssql19-
...
+#### Workaround
+If the performance of the first call is important, it is recommended to keep the queries warm. For example, a background task can be executed that fires a simple `sp_execute_external_script` query before the processes get expired. For instance, to keep R queries warm, you may execute the following query periodically.
+
+```sql
+EXECUTE sp_execute_external_script @language = N'R', @script = N'';
+GO
+```
+
+#### High number of concurrent queries
+If the number of concurrent execution of `sp_execute_external_script` is higher than the active R/Python processes in the pool, the cold start of adding additional processes to the pool may be slow (e.g., due to hardware constraints).
```
Comment (uc-msft) [inline docs/machine-learning/troubleshooting/known-issues-for-sql-server-machine-learning-services.md]: "hardware constraints" --> "resource constraints"

**Comment by arunguru-msft** [comment]
Looks good

File: docs/machine-learning/troubleshooting/known-issues-for-sql-server-machine-learning-services.md [hunk 2]
```diff
@@ -281,6 +281,29 @@ The best solution is to upgrade to [!INCLUDE[sssql19-md](../../includes/sssql19-
...
 > If you do the steps above, you must manually remove the added key prior to upgrading to a later version of SQL Server.
 
+## Performance issues of Process Pooling in ML Services (R and Python) 
+This section contains known issues and workarounds for using ML services (R and Python) in SQL Server.
+
+### Cold start Performance of Process Pooling in ML Services
+Upon execution of `sp_execute_external_script`, the launchpad service launches satellite processes that start the external runtimes such as R and Python. To amortize the startup cost, a pool of processes is created that can be used in the subsequent execution of `sp_execute_external_script`. This pool of processes is specific to this user, database, and the used language (R or Python in ML Services). 
+
+#### First query execution
+The satellite processes need to be warmed up when `sp_execute_external_script` is executed for the first time or after a period of idle time (the processes are terminated via a cleanup task if they are not used for a while). Cold start of such pooled processes may be slow (e.g., due to hardware constraints).
```
Comment (WilliamDAssafMSFT) [inline docs/machine-learning/troubleshooting/known-issues-for-sql-server-machine-learning-services.md]:
```suggestion
The satellite processes need to be warmed up when `sp_execute_external_script` is executed for the first time or after a period of idle time (the processes are terminated via a cleanup task if they are not used for a while). Cold start of such pooled processes may be slow (for example, due to resource constraints). 
```

File: docs/machine-learning/troubleshooting/known-issues-for-sql-server-machine-learning-services.md (hunk 1 above)
Comment (WilliamDAssafMSFT) [inline docs/machine-learning/troubleshooting/known-issues-for-sql-server-machine-learning-services.md]:
```suggestion
If the number of concurrent execution of `sp_execute_external_script` is higher than the active R/Python processes in the pool, the cold start of adding additional processes to the pool may be slow (for example, due to resource constraints).
```

**Review by WilliamDAssafMSFT — CHANGES_REQUESTED** [review]

File: docs/machine-learning/troubleshooting/known-issues-for-sql-server-machine-learning-services.md (hunk 1 above)
Comment (WilliamDAssafMSFT) [reply docs/machine-learning/troubleshooting/known-issues-for-sql-server-machine-learning-services.md]: updated

**Review by WilliamDAssafMSFT — APPROVED** [review]

**Pull request closed without merging**